File: src/main/scala/eschool/users/model/Guardian.scala
package eschool.users.model

import scala.collection.JavaConverters._

import javax.jdo.annotations._

@PersistenceCapable
class Guardian extends Perspective {
  private[this] var _children: java.util.Set[Student] = _
  
  def children: Set[Student] = _children.asScala.toSet
  def children_=(theChildren: Set[Student]) { _children = theChildren.asJava }
}

File: src/main/java/eschool/users/model/jdo/Guardian.java
package eschool.users.model.jdo;

import java.util.Set;
import javax.jdo.annotations.*;

@PersistenceCapable
public class Guardian extends Perspective {
  private Set<Student> children;
  
  public Set<Student> getChildren() {
	  return this.children;
  }
  
  public void setChildren(Set<Student> children) {
	  this.children = children;
  }
}
